lib/include/rpmsg_env.h

```diff
@@ -155,7 +156,16 @@ void env_memcpy(void *dst, void const *src, uint32_t len);
 int32_t env_strcmp(const char *dst, const char *src);
 void env_strncpy(char *dest, const char *src, uint32_t len);
 int32_t env_strncmp(char *dest, const char *src, uint32_t len);
+#ifdef MCUXPRESSO_SDK
```

Comment:
For me personaly this looks like it doesn't belongs here. We moved OS specific things into its header file. 
This looks like same approach should be used.

Reply:
I would keep it as is because this printf part is library-specific, not OS-specific.

Reply:
ok